pkg/registry/graph.go

```diff
@@ -21,14 +21,14 @@ type Node interface {
 	Name() string
 	// Type returns the type of the registry element a Node refers to
 	Type() Type
-	// AncestorNames returns a set of strings containing the names of all of the node's ancestors
+	// Ancestors returns a set of strings containing the names of all of the node's ancestors
 	Ancestors() []Node
-	// DescendantNames returns a set of strings containing the names of all of the node's descendants
+	// Descendants returns a set of strings containing the names of all of the node's descendants
 	Descendants() []Node
-	// ParentNames returns a set of strings containing the names of all the node's parents
+	// Parents returns a set of strings containing the names of all the node's parents
 	Parents() []Node
-	// ChildrenNames returns a set of strings containing the names of all the node's children
-	Childrens() []Node
+	// Children returns a set of strings containing the names of all the node's children
+	Children() []Node
```

Comment:
Might as well correct the comments, these haven't been sets of strings for a while.

Reply:
Oh, I'm late.  I'll fix it once this merges.